polaris-react/src/components/Filters/Filters.tsx

```diff
@@ -97,7 +97,7 @@ export interface FiltersProps {
   onAddFilterClick?: () => void;
   /** Whether the filter should close when clicking inside another Popover. */
   closeOnChildOverlayClick?: boolean;
-  /** The name of the currently selected view */
+  /** @deprecated The name of the currently selected view */
```

Comment:
Usually we offer an alternative instead but this may not be possible in this case. The message would read like:

`@deprecated Use title instead`

Reply:
@kyledurand Yeah, for this use case there's no alternative. It's only used to populate the suffix in the SearchField, and we are removing that functionality after key merchant frustrations. Good to know for any future deprecations though, thanks!